gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -202,6 +202,18 @@ typedef struct _GstTensorFilterFramework
        *
        * @param[in] data the data element.
        */
+
+  int (*updateModel) (const GstTensorFilterProperties * prop,
+      void **private_data, const char * model_path, const char * model_path_sub);
+      /**< Optional. tensor_filter.c will call it when it gets a related event.
+       * @todo add more detail comments.
+       *
+       * @param[in] prop read-only property values
+       * @param[in/out] private_data A subplugin may save its internal private data here. The subplugin is responsible for alloc/free of this pointer. Normally, close() frees private_data and set NULL.
+       * @param[in] model_path model path to update in runtime
+       * @param[in] model_path_sub sub model path to update in runtime
+       * @return 0 if ok. < 0 if error.
+       */
```

Comment:
So.. maybe..
```reloadModel (const GstTensorFilterProperties * prop, void **private_data);``` should be enough for this capability.```

Reply:
Got it. It'll be updated.